File: app/src/main/java/com/chris/takenote/data/pojo/Beauty.java
package com.chris.takenote.data.pojo;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class Beauty {
    @SerializedName("_id")
    public String id;

    @SerializedName("createdAt")
    public Date createTime;

    @SerializedName("desc")
    public String desc;

    @SerializedName("publishedAt")
    public Date publishTime;

    @SerializedName("source")
    public String source;

    @SerializedName("type")
    public String type;

    @SerializedName("url")
    public String imgUrl;

    @SerializedName("used")
    public boolean used;

    @SerializedName("who")
    public String author;
}

File: app/src/main/java/com/chris/eban/data/pojo/Beauty.kt
package com.chris.eban.data.pojo

import com.google.gson.annotations.SerializedName
import java.util.*

data class Beauty(
        @SerializedName("_id") var id: String?,
        @SerializedName("createdAt") var createTime: Date?,
        @SerializedName("desc") var desc: String?,
        @SerializedName("publishedAt") var publishTime: Date?,
        @SerializedName("source") var source: String?,
        @SerializedName("type") var type: String?,
        @SerializedName("url") var imgUrl: String?,
        @SerializedName("used") var used: Boolean,
        @SerializedName("who") var author: String?
)
